src/middleware/formData.ts

```diff
@@ -0,0 +1,58 @@
+//importar
+import busboy from "busboy";
+import { v2 as cloudinary } from "cloudinary";
+import { Request, Response, NextFunction } from "express";
+
+
+cloudinary.config({
+  cloud_name: "dmshyev5s",
+  api_key: "295928881237629",
+  api_secret: "gItinS6-WkwZP45dZnfulJtnino",
```

Comment:
These values for cloudinary should be better in constant variables or in env variables, for safety.